mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -98,6 +98,15 @@ class LogisticRegression(override val uid: String)
   def setFitIntercept(value: Boolean): this.type = set(fitIntercept, value)
   setDefault(fitIntercept -> true)
 
+  /**
+   * Whether to standardize the training features prior to fitting the model sequence.
```

Comment:
This is copied from R's description. I think it's about fitting a sequence of models with different regularization. I will modify it to `to fitting the model`.

```R
standardize	
Logical flag for x variable standardization, prior to fitting the model sequence. 
The coefficients are always returned on the original scale. Default is standardize=TRUE. 
If variables are in the same units already, you might not wish to standardize. 
See details below for y standardization with family="gaussian".
```